airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/SourceAcceptanceTest.java

```diff
@@ -77,7 +77,7 @@
    * FIXME: Some sources can't guarantee that there will be no events between two sequential sync
    */
   private Set<String> IMAGES_TO_SKIP_IDENTICAL_FULL_REFRESHES = Sets.newHashSet(
-      "airbyte/source-google-workspace-admin-reports");
+      "airbyte/source-google-workspace-admin-reports", "airbyte/source-kafka");
```

Comment:
Nitpick. Can you add a comment about why the kafka source can have extra events between two sequential sync?

Reply:
after the first poll() of the data, the offset will be committed and when the data is read again in the kafka it will no longer be, so two identical syncs will return different results

Reply:
Thanks. But can you add the comment to the code?